server.py

```diff
@@ -51,14 +51,16 @@ async def start_server():
     await web.TCPSite(runner, BaseWorld.get_config('host'), BaseWorld.get_config('port')).start()
 
 
-def run_tasks(services, start_vue_server=False):
+def run_tasks(services, run_vue_server=False):
     loop = asyncio.get_event_loop()
     loop.create_task(app_svc.validate_requirements())
     loop.run_until_complete(data_svc.restore_state())
     loop.run_until_complete(knowledge_svc.restore_state())
     loop.run_until_complete(RestApi(services).enable())
     loop.run_until_complete(app_svc.register_contacts())
     loop.run_until_complete(app_svc.load_plugins(args.plugins))
+    if run_vue_server:
+        loop.run_until_complete(app_svc.enable_cors())
```

Comment:
I'm sure this is placed here for a reason but wanted to double check -- could we move this to the `if` block on line 73?

Reply:
Yeah, it needs to be before the server is started and line 73 has to be after the server is started. So unfortunately they need to be separate if blocks